subsys/net/lib/lwm2m/lwm2m_message_handling.c

```diff
@@ -204,12 +204,9 @@ void lwm2m_engine_context_close(struct lwm2m_ctx *client_ctx)
 
 void lwm2m_engine_context_init(struct lwm2m_ctx *client_ctx)
 {
-	sys_slist_init(&client_ctx->pending_sends);
```

Comment:
`lwm2m_pull_context.c` also creates a `lwm2m_ctx` and makes use of `pending_sends`, IMO those initializers belong here. Perhaps `lwm2m_engine_context_init()` should not be called if the client does not start fresh but wakes up?

Reply:
OK I will check that also. context init was main key src for loose a buffered data. I will check lwm2m_pull_context.